bip-update-process.md

```diff
@@ -0,0 +1,646 @@
+substantial functional changes, it may be preferable to move it to Abandoned[^new-BIP], and to start a new BIP with the changes
+instead. Otherwise, it could cause confusion as to what being compliant with the BIP means.
+
+#### Active
+
+A settled[^settled] BIP may be advanced to Active upon request by
+any community member with evidence that the idea described in the BIP is in active use. Such evidence includes
+for example: two or more projects having deployed support for the BIP in mainnet software releases, a soft fork
+proposal’s activation criteria having been met on the network, or rough consensus for the BIP having been demonstrated.
+
+At that point, the BIP should be considered final and any breaking changes to the BIP should be proposed as a new
+separate BIP.[^new-BIP]
+
+##### Process BIPs
+
+A Process BIP may change status from Proposed to Active when it achieves rough consensus on the mailing list. Such a
+proposal is said to have rough consensus if it has been open to discussion on the development mailing list for at least
+one month, and no person maintains any unaddressed substantiated objections to it. Addressed or obstructive objections
+may be ignored/overruled by general agreement that they have been sufficiently addressed, but clear reasoning must be
+given in such circumstances. Active Process BIPs may be modified indefinitely as long as a proposed modification
+has rough consensus per the same criteria.[^living-documents]
+
+##### Revisions
+
+To help implementers understand updates to a BIP, any changes after it has reached Proposed or Active are
+tracked with version, date, and description in a Changelog section. The version number is inspired by semantic
+versioning (MAJOR.MINOR.PATCH). The MAJOR version is incremented if changes to the BIP’s Specification are introduced
+that are incompatible with prior versions (which should be an exception after a BIP is Proposed, and not happen to a
+BIP that is Active). The MINOR version is incremented whenever the specification of the BIP
+is changed or extended in a backward-compatible way. The PATCH version is incremented for other changes to the BIP that
+are noteworthy (bug fixes, test vectors, important clarifications, etc.). The version should start at 1.0.0 with the
+date the BIP was promoted to Proposed. An example of a Changelog section can be seen in
+[BIP 352](https://github.com/bitcoin/bips/blob/master/bip-0352.mediawiki). After a BIP receives a Changelog, the
+Preamble must indicate the latest version in the Revision header.
+
+#### Abandoned[^abandoned]
+
+A BIP that is _not in active use_, may be labeled Abandoned when its author has stopped working on it, no longer
+recommends the proposed approach, or no longer pursues adoption of their completed proposal. The reason for moving the
+proposal to Abandoned should be recorded in the Changelog section in the same commit that updates the status.
```

Comment:
```suggestion
A BIP that is _not in active use_ may be labeled Abandoned when its author has stopped working on it, no longer
recommends the proposed approach, or no longer pursues adoption of their completed proposal. The reason for moving the
proposal to Abandoned should be recorded in the Changelog section in the same commit that updates the status.
```

Isn't this paragraph just describing "Draft -> Abandoned" and "Proposed -> Abandoned" but not "Active -> Abandoned"

Reply:
I think it’s fine. "Not in active use" could be either achieved by _not yet_ being in active use, or _no longer_ being in active use. Happy to adjust if you feel it’s worth clarifying.

Reply:
I think my comment here was about the subsections. "Active -> Draft" is valid transition but it has no subsection mentioning it.

Reply:
I don’t see how "Active ↦ Draft" (now "Deployed" ↦ "Draft") would be a valid transition. In order for something to have been "Active/Deployed" it was adopted by the community and probably deployed on the network by at least one implementation. Returning to Draft for a significant shift in focus or deliberate breaking changes, would pull the Specification-rug out underneath of implementations deployed to the network. In that case it would be more appropriate to create another proposal, and to move the existing proposal to Closed when it is no longer active on the network.